spec/plurimath/math/formula/mathml_spec.rb

```diff
@@ -112,7 +112,7 @@
       let(:exp) {
         Plurimath::Math::Formula.new([
           Plurimath::Math::Function::Base.new(
-            Plurimath::Math::Function::Sin.new,
+            Plurimath::Math::Symbol.new("sin"),
```

Comment:
Err.. I'm not convinced that this is correct. Sin is **not** a symbol but a function.

Reply:
As noted above, I did not ask for this, this is a bug in LatexMath output which I had not noticed when I raised the ticket.